### PR TITLE
Allow for PR only edits

### PR DESCRIPTION
**Pull request by SimonBiggs**

### CODE REVIEW

File: src/documents/released/sop-document-record-control/index.md [hunk 1]
```diff
@@ -374,6 +378,9 @@ retention periods as outlined in this SOP.
...
 | Activity | Date       | Role                  | Name             | Initials |
 | -------- | ---------- | --------------------- | ---------------- | -------- |
-| Creation | 2021-02-24 | CEO                   | Simon Biggs      | SB       |
-| Review   | 2021-02-24 | Regulatory Consultant | Dr. Oliver Eidel | OE       |
-| Approval | 2021-02-28 | CEO                   | Simon Biggs      | SB       |
+| Creation | 2022-02-24 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-02-24 | Regulatory Consultant | Dr. Oliver Eidel | OE       |
+| Approval | 2022-02-28 | CEO                   | Simon Biggs      | SB       |
+| Changes  | 2022-02-28 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel |        |
```
Comment (SimonBiggs) [inline src/documents/released/sop-document-record-control/index.md:385]:
```suggestion
| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel |        |
```

File: src/documents/released/sop-document-record-control/index.md [hunk 2]
```diff
@@ -141,9 +141,9 @@ documents.
 
 | Activity | Date       | Role                  | Name             | Initials |
 | -------- | ---------- | --------------------- | ---------------- | -------- |
-| Creation | 2021-02-24 | CEO                   | Simon Biggs      | SB       |
-| Review   | 2021-02-24 | Regulatory Consultant | Dr. Oliver Eidel |          |
-| Approval | 2021-02-24 | CEO                   | Simon Biggs      |          |
+| Creation | 2022-02-24 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-02-24 | Regulatory Consultant | Dr. Oliver Eidel |          |
```
Comment (olieidel) [inline src/documents/released/sop-document-record-control/index.md:145]:
```suggestion
| Review   | 2022-02-24 | Regulatory Consultant | Dr. Oliver Eidel | OE       |
```

File: src/documents/released/sop-document-record-control/index.md (hunk 1 above)
Comment (olieidel) [inline src/documents/released/sop-document-record-control/index.md:385]:
```suggestion
| Review   | 2022-0 | Regulatory Consultant | Dr. Oliver Eidel | OE     |
```

**Review by olieidel — APPROVED** [review]

**Comment by SimonBiggs** [comment]
Thanks @olieidel :slightly_smiling_face:

File: src/documents/released/sop-document-record-control/index.md [hunk 3]
```diff
@@ -141,9 +141,9 @@ documents.
 
 | Activity | Date       | Role                  | Name             | Initials |
 | -------- | ---------- | --------------------- | ---------------- | -------- |
-| Creation | 2021-02-24 | CEO                   | Simon Biggs      | SB       |
-| Review   | 2021-02-24 | Regulatory Consultant | Dr. Oliver Eidel |          |
-| Approval | 2021-02-24 | CEO                   | Simon Biggs      |          |
+| Creation | 2022-02-24 | CEO                   | Simon Biggs      | SB       |
+| Review   | 2022-02-24 | Regulatory Consultant | Dr. Oliver Eidel | OE       |
```
Comment (SimonBiggs) [inline src/documents/released/sop-document-record-control/index.md:145]:
```suggestion
| Review   | 2022-02-24 | Regulatory Consultant | Dr. Oliver Eidel |          |
```

**Pull request merged**